locales-src/onboarding.ftl

```diff
@@ -69,6 +73,15 @@ onboarding-benefit-privacy-title = True Privacy
 # transparent about how we use it."
 onboarding-benefit-privacy-text = Everything we do honors our Personal Data Promise: Take less. Keep it safe. No secrets.
 
+onboarding-benefit-sync-title = Sync
+onboarding-benefit-sync-text = Take your bookmarks, passwords, history, and more everywhere you use { -brand-product-name }.
+
+onboarding-benefit-monitor-title = Monitor
+onboarding-benefit-monitor-text = Get notified when your personal info is in a data breach.
+
+onboarding-benefit-lockwise-title = Lockwise
```

Comment:
There's `-lockwise-brand-short-name = Lockwise` although not sure if we just do
`onboarding-benefit-lockwise-title = { -lockwise-brand-short-name }` vs directly reference the brand term (if that's even allowed ?)

Reply:
No, that's a prerogative of [terms](https://projectfluent.org/fluent/guide/terms.html).

> Terms are similar to regular messages but they can only be used as references in other messages